sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -713,6 +751,14 @@ export interface FreshnessScoringParameters {
     boostingDuration: string;
 }
 
+// @public (undocumented)
```

Comment:
we should document these if they're public facing

Reply:
Done!